humble/distribution.yaml

```diff
@@ -5235,6 +5235,16 @@ repositories:
       url: https://github.com/Simple-Robotics/proxsuite.git
       version: devel
     status: developed
+  psdk_ros2:
+    doc:
+      type: git
+      url: https://github.com/umdlife/psdk_ros2
```

Comment:
```suggestion
      url: https://github.com/umdlife/psdk_ros2.git
```

Reply:
Resolved in https://github.com/ros/rosdistro/pull/39574/commits/81e1e3e30076363e5c459ab6bcead4407e6d3a08